C-18-01-Capstone.Web/Controllers/UserController.cs

```diff
@@ -49,7 +49,7 @@ public ActionResult Register()
             var dataAccess = new DataAccess<Country>();
 
             HtmlLists.Countries = dataAccess.GetEntities().OrderBy(_ => _.Name).ToList();
```

Comment:
In file above you are filling this field and here you are overwriting it again. It is a mistake